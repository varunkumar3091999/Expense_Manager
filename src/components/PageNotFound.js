import React from 'react'
import { BrowserRouter, Route, Switch, Link, NavLink } from 'react-router-dom'


const PageNotFound = () => (
  <div>
    this is a 404 component
    <Link to='/'>go home</Link>
  </div>
)

export default PageNotFound