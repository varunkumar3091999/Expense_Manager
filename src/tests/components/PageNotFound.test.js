import React from 'react';
import { shallow } from "enzyme";
import PageNotFound from '../../components/PageNotFound';


test(
  "should render 404 page correctly",
  () => {
    const wrapper = shallow(<PageNotFound />)
    expect(wrapper).toMatchSnapshot()
  }
)