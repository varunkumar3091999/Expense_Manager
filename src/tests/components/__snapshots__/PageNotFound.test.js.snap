// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 404 page correctly 1`] = `
<div>
  this is a 404 component
  <Link
    to="/"
  >
    go home
  </Link>
</div>
`;
